builder/image/templates/builder

```diff
@@ -101,7 +101,7 @@ if [ ! -f Dockerfile ]; then
     if [ -f /buildpacks ]; then
         BUILD_OPTS+=' -v /buildpacks:/tmp/buildpacks:rw'
         # give non-root slugbuilder user R/W perms for docker volumes
-        chown -R 2000:2000 /buildpacks
+        sudo chown -R 2000:2000 /buildpacks
```

Comment:
It has to be done at container startup (i think?), so it would require changing the entrypoint in slugbuilder to a new script - executed as root. This script would `chown` and then execute the current `builder/build.sh` script then as the `slug` user.